.ci/Jenkinsfile-tests

```
@@ -22,6 +22,7 @@ pipeline {
       INMANTA_LS_PATH="${env.WORKSPACE}/server"
       INMANTA_LS_LOG_PATH="${env.WORKSPACE}/server.log"
       PIP_INDEX_URL="https://artifacts.internal.inmanta.com/inmanta/dev"
+      INMANTA_STYLE_CHECK_ENV="./style-check-venv"
```

This ensures an absolute path is being used. That will prevent issues when the cwd is being updated.
```suggestion
      INMANTA_STYLE_CHECK_ENV="${env.WORKSPACE}/style-check-venv"
```